scripts/items/kupo_rod.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+-- ID: 21074
+-- Item: Kupo Rod
+-- Enchantment: Moogle Blessing (removes Weakness)
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target)
+    return 0
```

Comment:
Can this item be used if the target is not weakened?

Reply:
ive added a check for this